setup.py

```diff
@@ -29,11 +31,15 @@
     ],
     keywords='dataset data iteration pipeline processing',
     packages=find_packages(exclude=['tests']),
-    install_requires=['six', 'picklable_itertools', 'pyyaml', 'h5py',
+    install_requires=['six', 'picklable_itertools', 'pyyaml', 'h5py', 'cython',
                       'tables', 'progressbar2', 'pyzmq', 'scipy', 'pillow',
                       'requests'],
     extras_require={
         'test': ['nose', 'nose2', 'mock']
     },
-    scripts=['bin/fuel-convert', 'bin/fuel-download', 'bin/fuel-info']
+    scripts=['bin/fuel-convert', 'bin/fuel-download', 'bin/fuel-info'],
+    ext_modules=cythonize(Extension("fuel.transformers._image",
```

Comment:
Alright, then we should add a note in #104 so we mention that in the installation docs, but other than that it's fine by me as well.